frontend/projects/abtesting/src/app/features/dashboard/home/components/modal/new-experiment/new-experiment.component.ts

```diff
@@ -57,18 +58,22 @@ export class NewExperimentComponent implements OnInit {
         }
         break;
       case NewExperimentDialogEvents.UPDATE_EXPERIMENT:
+        this.onNoClick();
       case NewExperimentDialogEvents.SAVE_DATA:
         this.newExperimentData = {
           ...this.experimentInfo,
           ...this.newExperimentData,
           ...formData
         };
+        this.openSnackBar();
         this.experimentService.updateExperiment(this.newExperimentData);
-        this.onNoClick();
         break;
     }
   }
 
+  openSnackBar() {
+    this._snackBar.open(`Experiment changes are updated successfully!`, null, { duration: 2000 })
```

Comment:
Can you please take this message from the en.json file?

Reply:
done